.gitignore

```diff
@@ -0,0 +1,13 @@
+**/__pycache__
+env/
+venv/
+.env
+.idea
+.pytest_cache/
+.mypy_cache/
+bin/
+lib/
+lib64
+**/$*.py
+apiadmin.egg-info/
```

Comment:
I think you meant to add '*.egg-info/' but it got expanded by your shell to some irrelevant project.